src/test/java/org/sonarlint/intellij/trigger/SonarDocumentListenerTest.java

```diff
@@ -37,11 +41,7 @@
 import org.sonarlint.intellij.util.SonarLintAppUtils;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.timeout;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyZeroInteractions;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Have a look at Code style -> java -> Imports -> Packages to use import with * (should be empty)